contracts/VaultV3.vy

```diff
@@ -606,24 +578,33 @@ def _process_report(strategy: address) -> (uint256, uint256):
     else:
         loss = current_debt - total_assets
 
+    # We compute unlocked_profit
+    profit_end_date: uint256 = self.profit_end_date
```

Comment:
OPTIMISATION: 
probably a good idea to do this inside the ` if self.profit_distribution_rate_ != 0:` clause. That way, if you have profit_distribution_rate = 0, you don't read `profit_end_date`, saving some gas

later in this function (L592) you need `profit_end_date`, but that also changes only if `profit_distribution_rate` is > 0. you would need `pending_profit` initialised to 0 here though

Reply:
Done! Good catch!